docs/apache-airflow-providers-postgres/connections/postgres.rst

```diff
@@ -29,7 +29,7 @@ Host (required)
     The host to connect to.
 
 Schema (optional)
-    Specify the schema name to be used in the database.
+    Specify the name of the database to connect to. Note that a database schema is not allowed during connect. Instead you can use ``SET search_path TO custom_schema`` if you want to change the default schema being used in a session for your queries. If you are using ``PostgresOperator`` see :ref:`Passing Server Configuration Parameters into PostgresOperator <howto/operators:postgres>`.
```

Comment:
One comment here. I think it needs a bit more explanation here. What does it mean for the user that "database schema is not allowed during connect"  - I think it should be described a bit more from the user perspective: "If you do not use Postgres operator but run query directly via Postgres Hook, you need also to set  schema via "SET search_path". ... Or comething like that (if I understand correctly the consequences)

Reply:
I have updated the docs in 8e9b1ba to reference postgres docs. I think this makes it clearer & cleaner. WDYT?

Reply:
Looks better :)